modular_zubbers/code/modules/autolathe_components_adv/advpartdisks.dm

```diff
@@ -0,0 +1,146 @@
+/datum/design/ultra_micro_laser
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/super_matter_bin
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/hyper_cell
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/hyper_battery
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+
+/obj/item/disk/design_disk/bluespace_parts
+	name = "Bluespace components autolathe design disk"
+
+/obj/item/disk/design_disk/bluespace_parts/Initialize(mapload)
+	. = ..()
+	blueprints += new /datum/design/quadratic_capacitor
+	blueprints += new /datum/design/triphasic_scanning
+	blueprints += new /datum/design/femto_servo
+	blueprints += new /datum/design/quadultra_micro_laser
+	blueprints += new /datum/design/bluespace_matter_bin
+	blueprints += new /datum/design/bluespace_cell
+	blueprints += new /datum/design/bluespace_battery
+
+/datum/design/quadratic_capacitor
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/triphasic_scanning
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/femto_servo
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/quadultra_micro_laser
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/bluespace_matter_bin
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/bluespace_cell
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/bluespace_battery
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+
+/datum/design/adv_part_disk
+	name = "Advanced components design disk"
+	desc = "A disk for an autolathe containing advanced component designs."
+	id = "adv_part_disk"
+	build_type = PROTOLATHE | AWAY_LATHE
+	materials = list(/datum/material/iron = SMALL_MATERIAL_AMOUNT*1.5)
+	build_path = /obj/item/disk/design_disk/advanced_parts
+	category = list(
+		RND_CATEGORY_STOCK_PARTS + RND_SUBCATEGORY_STOCK_PARTS_2
+	)
+	lathe_time_factor = 0.2
+	departmental_flags = DEPARTMENT_BITFLAG_ENGINEERING | DEPARTMENT_BITFLAG_SCIENCE | DEPARTMENT_BITFLAG_CARGO | DEPARTMENT_BITFLAG_MEDICAL
+
+/datum/design/high_tech_part_disk
+	name = "High-tech components design disk"
+	desc = "A disk for an autolathe containing high-tech component designs."
+	id = "high_tech_part_disk"
+	build_type = PROTOLATHE | AWAY_LATHE
+	materials = list(/datum/material/iron = SMALL_MATERIAL_AMOUNT*1.5)
+	build_path = /obj/item/disk/design_disk/high_tech_parts
+	category = list(
+		RND_CATEGORY_STOCK_PARTS + RND_SUBCATEGORY_STOCK_PARTS_2 + RND_SUBCATEGORY_STOCK_PARTS_3
+	)
+	lathe_time_factor = 0.2
+	departmental_flags = DEPARTMENT_BITFLAG_ENGINEERING | DEPARTMENT_BITFLAG_SCIENCE | DEPARTMENT_BITFLAG_CARGO | DEPARTMENT_BITFLAG_MEDICAL
+
+/datum/design/bluepspace_part_disk
+	name = "Bluespace components design disk"
+	desc = "A disk for an autolathe containing bluespace component designs."
+	id = "bluespace_part_disk"
+	build_type = PROTOLATHE | AWAY_LATHE
+	materials = list(/datum/material/iron = SMALL_MATERIAL_AMOUNT*1.5)
```

Comment:
I'm not going to be super strict on the material cost because arcmining exists, but I feel like there should be some tech level material cost. Diamond and BSCs for T4. Gold and silver for T3, etc